### ISSUE TITLE
Segfault on .permute() call with identity transform

### ISSUE THREAD

**Issue by memetb** [pytorch/pytorch#1480]
```python
import torch
from torch.autograd import Variable
import torch.nn as nn

conv = nn.Conv1d( 512, 32, 5, padding=2 )

# this works
input = Variable( torch.randn( 120, 512 , 1 ), volatile=True)
output = conv(input).permute(0,1,2)

# this segfaults
input = Variable( torch.randn( 120, 512 , 1 ))
output = conv(input).permute(0,1,2) # <- segfault
```

torch version 0.1.11+27fb875 and 0.1.12_2
Python 3.6.0 |Anaconda 4.3.1 (64-bit)

**Comment by lantiga** [comment]
@apaszke This issue should be solved now that #1493 is merged.

**Comment by apaszke** [comment]
You're right. Thanks! 🙂
